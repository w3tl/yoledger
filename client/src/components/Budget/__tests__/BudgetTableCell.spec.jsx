import React from 'react';
import { mount } from 'enzyme';
import { Table } from 'semantic-ui-react';
import RawBudgetTableCell from '../BudgetTableCell';

const BudgetTableCell = props => (
  <Table>
    <Table.Body>
      <Table.Row>
        <RawBudgetTableCell {...props} />
      </Table.Row>
    </Table.Body>
  </Table>
);

describe('BudgetTableCell', () => {
  it('should call update when blur', () => {
    const onUpdate = jest.fn();
    const output = mount(
      <BudgetTableCell onUpdate={onUpdate} amount={5} />,
    );
    output.find('AmountInput').simulate('blur');
    expect(onUpdate.mock.calls[0]).toMatchSnapshot();
  });
});
