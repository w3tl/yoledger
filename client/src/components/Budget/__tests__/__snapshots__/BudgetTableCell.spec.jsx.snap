// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BudgetTableCell should call update when blur 1`] = `undefined`;
